quickstarts/multicodebase-hellos/README.md

```diff
@@ -0,0 +1,49 @@
+# Firebase SDK for Cloud Functions Quickstart - Multi-codebases
+
+This quickstart demonstrates how to setup a multi-codebase repository that allows you to setup multiple source directories to manage your function definitions. 
```

Comment:
When it's a verb, it needs the space: "set up," here and the next instance (the following heading is fine as it's presumably a noun).

Important internal reference: https://memegen.corp.google.com/6809714